src/commands/whoami.js

```diff
@@ -0,0 +1,51 @@
+const Command = require('../base')
+const globalFlags = require('../helpers/global-flags')
+const authHelper = require('../helpers/auth')
+const requestHelper = require('../helpers/request')
+const ora = require('ora')
+const chalk = require('chalk')
+
+/**
+ * Tells you who you're logged in as
+ */
+class WhoamiCommand extends Command {
+  async run() {
+    if (!authHelper.isLoggedIn()) {
+      return this.log(`Not sure, you aren't logged in yet. Run ${chalk.yellow('chec login')} to continue.`)
+    }
+
+    const spinner = ora({
+      text: 'Processing...',
+      stream: process.stdout,
+    }).start()
+
+    const {flags: {domain}} = this.parse(WhoamiCommand)
+
+    try {
+      const result = await requestHelper.request('GET',
+        '/v1/developer/user', {}, {domain})
+      spinner.stop()
+
+      const data = JSON.parse(result.body)
+      this.log(`${chalk.green.bold(data.email)}: ${data.merchants.map(merchant => merchant.business_name).join(', ')}`)
+    } catch (error) {
+      spinner.fail('Request failed! ' + error.statusCode + ' ' + error.statusMessage)
```

Comment:
Do you want to explicitly capture a `403` because of revoked/changed keys here? Might be able to give a nicer message in this case? Eg. "You have been logged out. Run `chec login` to log back in".

Reply:
Wouldn't that apply to all requests?

Reply:
Hmmmn. I suppose it would. Could be a global handler then? Maybe worth addressing separately?

Reply:
I'll raise an issue to discuss it

Reply:
#19